dynamixel_port:
  port_name: "/dev/input/dx_upper"
  baud_rate: 3000000
  joints:
    - head_pan_joint
    - head_tilt_joint
    - wheel_f_l_steer_joint
    - wheel_f_r_steer_joint
    - wheel_b_l_steer_joint
    - wheel_b_r_steer_joint
    - wheel_f_l_drive_joint
    - wheel_f_r_drive_joint
    - wheel_b_l_drive_joint
    - wheel_b_r_drive_joint

  head_pan_joint:               {id: 21, center: 2048, home:  0.00, mode: 3, vel: 200, acc: 0, pos_d_gain:    0, pos_i_gain:  0, pos_p_gain:  800, gear_ratio: 1 }
  head_tilt_joint:              {id: 22, center: 2048, home:  0.00, mode: 3, vel: 200, acc: 0, pos_d_gain:    0, pos_i_gain:  0, pos_p_gain:  800, gear_ratio: 1 }

  wheel_f_l_steer_joint: {id: 5, center: 2048, home:  0.00, mode: 3, vel: 200, acc: 0, pos_d_gain:    0, pos_i_gain:  0, pos_p_gain:  800, gear_ratio: 1 }
  wheel_f_r_steer_joint: {id: 6, center: 2048, home:  0.00, mode: 3, vel: 200, acc: 0, pos_d_gain:    0, pos_i_gain:  0, pos_p_gain:  800, gear_ratio: 1 }
  wheel_b_l_steer_joint: {id: 7, center: 2048, home:  0.00, mode: 3, vel: 200, acc: 0, pos_d_gain:    0, pos_i_gain:  0, pos_p_gain:  800, gear_ratio: 1 }
  wheel_b_r_steer_joint: {id: 8, center: 2048, home:  0.00, mode: 3, vel: 200, acc: 0, pos_d_gain:    0, pos_i_gain:  0, pos_p_gain:  800, gear_ratio: 1 }
  
  wheel_f_l_drive_joint: {id: 1, center: 2048, home:  0.00, mode: 3, vel: 200, acc: 0, pos_d_gain:    0, pos_i_gain:  0, pos_p_gain:  800, gear_ratio: 1 }
  wheel_f_r_drive_joint: {id: 2, center: 2048, home:  0.00, mode: 3, vel: 200, acc: 0, pos_d_gain:    0, pos_i_gain:  0, pos_p_gain:  800, gear_ratio: 1 }
  wheel_b_l_drive_joint: {id: 3, center: 2048, home:  0.00, mode: 3, vel: 200, acc: 0, pos_d_gain:    0, pos_i_gain:  0, pos_p_gain:  800, gear_ratio: 1 }
  wheel_b_r_drive_joint: {id: 4, center: 2048, home:  0.00, mode: 3, vel: 200, acc: 0, pos_d_gain:    0, pos_i_gain:  0, pos_p_gain:  800, gear_ratio: 1 }
